### PR TITLE
Revert app name back to Simplenote and fix newline issue in AppStoreStrings.pot

### PR DESCRIPTION
**Pull request by oguzkocer**
We have a request from @SylvesterWilmott to revert the app name back to Simplenote, so I've updated `fastlane/appstoreres/metadata/source/name.txt` with that, regenerated the `AppStoreStrings.pot` file and manually fixed the newline issue.

@AliSoftware I assume this will still import the `Simplenote` app name to GlotPress and I was wondering if that's something we still want. I can imagine a case where someone would translate it even though we don't want it to be translated anymore. Do you think we should just stop uploading the app name to GlotPress? If so, could you help me make that change (or open a PR against the release branch)?

### CODE REVIEW

**Comment by AliSoftware** [comment]
> We have a request from @SylvesterWilmott to revert the app name back to Simplenote

Ref for info/tracking: paaHJt-2Fl-p2

**Comment by AliSoftware** [comment]
> @AliSoftware I assume this will still import the `Simplenote` app name to GlotPress and I was wondering if that's something we still want. I can imagine a case where someone would translate it even though we don't want it to be translated anymore. Do you think we should just stop uploading the app name to GlotPress? If so, could you help me make that change (or open a PR against the release branch)?

That's a good question.

I think what you could do is revert the relevant changes made in https://github.com/Automattic/simplenote-ios/pull/1411/files, especially:
 - Remove the entry from the `pot` file – which will mark the key as deleted in GlotPress, avoiding the risk to have people translate it into something fancy;
 - Remove the `app_store_title:` line on line 114 of the `Fastfile` (so it doesn't re-insert that entry in the `pot` from the `name.txt` file next time you re-generate it)
 - Revert the changes in `download_metadata.swift`, so that we won't try to download the app title from GlotPress; or even better, just `//` comment [this line](https://github.com/Automattic/simplenote-ios/pull/1411/files#diff-cdd9e7b5c9c1c56db9677d01e281315f16469b529a0d8d4c3992e5f2a4844b4eR111) – so that it will be easier to restore when time comes, and also to ensure you still keep the fix about `subtitle` being mapped to the correct `.txt` file (line 104/112 of that diff)
 - Update/`cp` all the `fastlane/metadata/<locale>/name.txt` files to all have the same content as `fastlane/appstoreres/metadata/source/name.txt` – since those will be used by `fastlane deliver` to update the metadata in ASC for you, but since with the previous points we won't update those txt files with GlotPress content anymore, better be sure they use the untranslated title in order for the AppStore to be in sync there.

Alternatively you could only do the last 2 points (which would keep the title key in GlotPress… but just ignore it during the download of the metadata and not update `name.txt` files anymore), but also doing the first 2 points will make the whole thing probably more consistent (ie not just the "pull from GP" part but also the "push" part).

File: fastlane/metadata/id/name.txt [hunk 1]
```diff
@@ -1 +0,0 @@
-Simplenote - Catatan dan Tugas
```
Comment (AliSoftware) [inline fastlane/metadata/id/name.txt:1]: Be sure to manually revert the copy in ASC for that `id` locale – as removing the file here means that `fastlane deliver` will just not touch/update the app name field for that locale in ASC anymore, but it was likely already updated in ASC during last release.

(I might be wrong here, i.e. maybe instead `fastlane deliver` clears the name for those locales in ASC to make them falls back to the English copy for locales which don't have a `name.txt` file, as opposed to leave them untouched? But better double-check anyway)

**Review by AliSoftware — APPROVED** [review]
👍  :shipit:

File: fastlane/metadata/id/name.txt (hunk 1 above)
Comment (oguzkocer) [reply fastlane/metadata/id/name.txt:1]: I've been manually removing the change each release waiting on the result/decision about the title translations. This name never made it to any of the releases, so we are all good.

**Pull request merged**